### ISSUE TITLE
about dropout=0.95

### ISSUE THREAD

**Issue by shuangshuangshuangfeng** [osmanberke/Deep-SSVEP-BCI#1]
Hello researcher:
I want to ask you why you use dropout=0.95 before the output layer. We all know that most people choose dropout=0.5. If it is too large, a lot of important information will be lost. I also tested dropout=0.95 and the results will be very good.  I want to know why you use dropout=0.95?

**Comment by osmanberke** [comment]
Hello,
In our structure, at the output layer, there are too many parameters (e.g. for signal duration 0.2 seconds, the number of parameters at the output layer is 25x120x40 + 40 = 120.040). And the number of training data is relatively small (7000 at benchmark dataset, 8400 at BETA dataset) that is why we need to have such higher dropout probability at that layer.
